lib/src/workflows/ensure_cache.workflow.dart

```diff
@@ -36,7 +37,7 @@ Future<CacheFlutterVersion> ensureCacheWorkflow(
         );
       }
 
-      if (integrity == CacheIntegrity.versionMismatch) {
+      if (integrity == CacheIntegrity.versionMismatch && !force && !validVersion.isCustom) {
```

Comment:
This makes sense, I suggest that we add a warning in case the step is ignored due to the force flag. We can use `logger.warn()` to notify the user that the step is being skipped because of the force flag, this just helps notify the user in case that was a side effect of the force flag.

Reply:
Good suggestion, I added some warns